code/datums/mob_stats.dm

```diff
@@ -210,7 +213,10 @@
 			. += SM.value
 
 /datum/stat/proc/setValue(value)
-	src.value = value
+	if(value > STAT_VALUE_MAXIMUM)
+		src.value = STAT_VALUE_MAXIMUM
+	else
+		src.value = value
```

Comment:
```suggestion
		value = STAT_VALUE_MAXIMUM
	else
		value = value
```

Reply:
why not just use CLAMP

Reply:
You mean `min()`?

Reply:
because i feel fancy

Reply:
I'm guessing there should be a minimum stat value as well.

Reply:
Stats can go negative, that's ok and intended, as far as I know.

Reply:
If someone was able to reliably lower their stats you could probably get fucky behavior as well.